sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -252,6 +216,42 @@ export enum ReceiveMode {
     receiveAndDelete = 2
 }
 
+// @public
+export interface Receiver<ContextT> {
+    close(): Promise<void>;
+    diagnostics: {
+        peek(maxMessageCount?: number): Promise<ReceivedMessage[]>;
+        peekBySequenceNumber(fromSequenceNumber: Long_2, maxMessageCount?: number): Promise<ReceivedMessage[]>;
+    };
+    // (undocumented)
```

Comment:
Can you copy the docs for these methods from the deleted `ServidceBusReceiverClient`(still present in master), and generate the report again!